alvr/client_core/cpp/gamma_pass.cpp

```diff
@@ -17,10 +17,23 @@ const string PASSTHROUGH_FRAGMENT_SHADER = R"glsl(#version 300 es
         uniform samplerExternalOES tex0;
         in vec2 uv;
         out vec4 color;
+
+        const float div12 = 1. / 12.92;
+        const float div1 = 1. / 1.055;
+        
+        float srgbToLinear(float val)
```

Comment:
I'm not familiar with shaders, so I might have stupid questions here. But is it possible to apply swizzle here and below? So it will be like:
```
float srgbToLinear(vec3 rgb)
...

color = texture(tex0, uv);
color.rgb = srgbToLinear(color.rgb);
```

Reply:
this would check the combined rgb value against the threshold afaik, we need to check them per channel